dff/messengers/discord/interface.py

```diff
@@ -0,0 +1,63 @@
+from io import BytesIO
+from logging import getLogger
+
+from dff.messengers.common.modules import discord
+from pydantic import HttpUrl
+
+from dff.messengers.common import CallbackMessengerInterface
+from dff.pipeline.types import PipelineRunnerFunction
+from dff.script.core.message import Attachments, Audio, Document, Image, Message, Video
+
+logger = getLogger(__name__)
+
+
+def extract_message_from_discord(message: discord.Message) -> Message:  # pragma: no cover
+    inn_mess = Message()
+    inn_mess.text = message.content
+
+    files = list()
+    if len(message.attachments) > 0:
+        first_attachment = message.attachments[0]
+        if first_attachment.content_type is not None:
+            content_type = first_attachment.content_type.split("/")[0]
+            if content_type == "audio":
+                files = [Audio(source=HttpUrl(first_attachment.url))]
+            elif content_type == "video":
+                files = [Video(source=HttpUrl(first_attachment.url))]
+            elif content_type == "image":
+                files = [Image(source=HttpUrl(first_attachment.url))]
+            elif content_type in ("application", "text"):
+                files = [Document(source=HttpUrl(first_attachment.url))]
+
+    inn_mess.attachments = Attachments(files=files)
```

Comment:
I don't understand what happened to other attachments (only the first one is added afaict). If this is intentional, add comments explaining why.

Reply:
It was intentional, since I wanted to match [this code](https://github.com/deeppavlov/dialog_flow_framework/blob/18908d026b03ada0393c694691c974d39872a91f/dff/messengers/telegram/interface.py#L58-L77) from `messengers/telegram/interface.py`.
As far as I understand, in `telegram` only one (last) attachment ends up in `TelegramMessage.update` field. If that is not correct or should be changed - let me know, I will change it both here and in `telegram` (when I will be updating it).

Reply:
Yes, I just checked tg bot -- it appears that every attachment is considered a separate update (user request).

That is not correct behavior and should be changed. Sending multiple images at once should be considered one user request.